test/clj/rems/test_performance.clj

```diff
@@ -0,0 +1,94 @@
+(ns ^:integration rems.test-performance
+  (:require [clj-memory-meter.core :as mm]
+            [medley.core :refer [map-vals]]
+            [mount.core :as mount]
+            [rems.api.applications-v2 :as applications-v2]
+            [rems.api.reviews :as reviews]
+            [rems.db.dynamic-roles :as dynamic-roles]))
+
+(defn duration-millis [f]
+  (let [start (System/nanoTime)
+        _ (f)
+        end (System/nanoTime)]
+    (/ (- end start) 1000000.0)))
+
+(defn run-benchmark [benchmark iterations]
+  ((:setup benchmark))
+  (doall (for [_ (range iterations)]
+           (duration-millis (:benchmark benchmark)))))
+
+(defn statistics [measurements]
+  (let [measurements (sort measurements)]
+    {:min (first measurements)
+     :median (nth measurements (quot (count measurements) 2))
+     :max (last measurements)}))
```

Comment:
instead of manually iterating and collecting statistics, consider a library like https://github.com/hugoduncan/criterium